frontend/rolecall/src/app/common_components/empty_string_if_undefined.pipe.ts

```diff
@@ -1,13 +1,13 @@
 import {Pipe, PipeTransform} from '@angular/core';
-import {isNullOrUndefined} from 'util';
+//import {isNullOrUndefined} from 'util';
 
 @Pipe({
   name: 'emptyStringIfUndefined'
 })
 export class EmptyStringIfUndefinedPipe implements PipeTransform {
 
   public transform(value: unknown): unknown {
-    if (isNullOrUndefined(value)) {
+    if (!value) {
```

Comment:
Shouldn't this be `!!`? I think currently this change caused the opposite logic.

Reply:
I think it is OK.
Originally, if value was null, the if statement would execute.
The new code, if value is null, the if statement will execute.

Reply:
Ah got it.